src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -868,6 +868,13 @@ public void commitOffset(Map<String, ?> offset) {
 
                     // Parse the string to get the OpId object.
                     OpId tempOpId = OpId.valueOf((String) entry.getValue());
+                    // Check if the received OpId is less than the checkpoint already present in the map.
```

Comment:
Code and the comment is same! Doesn't add any value.
Add the context, like if we checkpoint lesser index, then we will get GCed error. And point to the issue.

Reply:
Resolved